draft-demarco-oauth-status-assertions.md

```diff
@@ -126,7 +126,7 @@ credentials, whether in JSON Web Tokens (JWT) or CBOR Web Tokens (CWT)
 format. Status Assertions function
 similarly to OCSP Stapling ([RFC6066]), allowing clients to present to the
 relying parties
-time-stamped assertions provided by the Issuer.
+time-stamped assertions provided by the issuer.
```

Comment:
@marinaado this is because we don't use defined terms representations before the section that defines them